README.md

```diff
@@ -1 +1,104 @@
-README
+<p align="center">
+  <h1 align="center">🧪👨‍💻 Jest Testing</h1>
+
+<h3 align='center'>  
+    A Jest Testing The Basics Project
+</h3>
+<p align="center">
+    <a href="https://github.com/Rhelli/Jest-Testing/archive/develop.zip">Download this repository</a>
+    |
+    <a href="https://github.com/Rhelli/Jest-Testing/issues/new">Report A Bug</a>
+    |
+    <a href="https://github.com/Rhelli/Jest-Testing/fork">Fork It</a>
+  </p>
+</p>
+
+
+## Table of Contents
+  - [The Project Brief](https://github.com/Psiale/todolist#-the-project-brief)
+  - [Project Specific](https://github.com/Psiale/todolist#-the-project-specific)
+  - [Tools & Tech](https://github.com/Psiale/todolist#-tools--technology-used)
+  - [Setup & Use](https://github.com/Psiale/todolist#-setup--use)
+  - [Contributions, Issues and Forking](https://github.com/Psiale/todolist#-contributions-issues-and-forking)
+  - [Creators](https://github.com/Psiale/todolist#-%EF%B8%8F-creators)
+  - [Show Your Support!](https://github.com/Psiale/todolist#-show-your-support)
+
+
+## 🧬 The Project Brief
+
+<strong>This project purpose was to learn the basics of Jest Testing by completing 5 different challengues  </strong>
+
+This is a project to be included by all students at [Microverse](https://www.microverse.org/) as part of the Javascript module.
+
+
+## 💡 The Project Specific
+Having just read about testing in Javascript in this project we learn how to implement our reading in the form of Jest testing.
+In this project we all given the objective to create five functions and create five separate test scripts for those functions, these functions include: 
+ - Capitalize: Capitalises a string
+ - Reverse string: Reverses a string (amazing!) 
+ - Calculator: Returns an object with the basic operations add, subtract, multiply, divide
+ - [Caesar-cypher](http://practicalcryptography.com/ciphers/caesar-cipher/): Takes a string and shifts letters up by the given number 
+ - Array analysis: Takes an array of numbers and returns an object with average, min, max, and length.
+
+## 🔬 Tools & Technology Used
+**Languages & Frameworks**
+ - Javascript
+ - Webpack 4.44
+ - Jest testing library 26.6.1
+
+**Package Management**
+ - NPM
+
+**Linters & Quality Control**
+ - [ESlint](https://eslint.org/)
+ - [StyleLint](https://stylelint.io/)
+
+### 💾 Setup & Use
+
+#### Setup
+For the live site, [head here.](https://mtodo-list.netlify.app/)
```

Comment:
- [ ] Please remove this live link on your readme. This is a testing project therefore it shouldn't have one.